src/NoLicense/NoLicense.js

```diff
@@ -0,0 +1,38 @@
+import { Box, Typography } from "@material-ui/core";
+import LockIcon from "@material-ui/icons/Lock";
+import PropTypes from "prop-types";
+import React from "react";
+
+export default function NoLicense({ labelHeader, labelBody }) {
+  // return components
+  return (
+    <Box height="100%" width="100%" display="flex">
+      <Box display="flex" flexDirection="column" alignItems="centre">
+        <LockIcon color="action" fontSize="medium" />
+      </Box>
+      <Box display="flex" flexDirection="column">
+        <Typography color="textSecondary" display="inline" variant="h6">
+          {labelHeader}
+        </Typography>
+        {labelHeader && labelHeader.length > 0 ? (
+          <Typography color="textSecondary" display="inline" variant="body1">
+            {labelBody}
+          </Typography>
+        ) : null}
+      </Box>
+    </Box>
+  );
+}
+
+// prop types
+NoLicense.propTypes = {
+  /**
+   * Message to display below the header text
+   */
+  labelBody: PropTypes.string,
+
+  /**
+   * Message to diplay alongside the icon (header)
+   */
+  labelHeader: PropTypes.string
+};
```

Comment:
I think it would be good if we could try to stick to single word prop names. Maybe instead of labelHeader and labelBody we could use title and description.

Reply:
done